great-idea-website/css/index.css

```
@@ -1,66 +1,308 @@
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
 }
 body {
-	line-height: 1;
+  line-height: 1;
 }
-ol, ul {
-	list-style: none;
+ol,
+ul {
+  list-style: none;
 }
-blockquote, q {
-	quotes: none;
+blockquote,
+q {
+  quotes: none;
 }
-blockquote:before, blockquote:after,
-q:before, q:after {
-	content: '';
-	content: none;
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
 }
 table {
-	border-collapse: collapse;
-	border-spacing: 0;
+  border-collapse: collapse;
+  border-spacing: 0;
 }
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+  box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
-    font-family: 'Titillium Web', sans-serif;
+html,
+body {
+  height: 100%;
+  font-family: "Titillium Web", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: "Bangers", cursive;
+  letter-spacing: 1px;
+  margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+p {
+  font-family: "Roboto", sans-serif;
+}
+
+/* header {
+	position: fixed;
+	z-index: 10;
+} */
+
+header {
+  margin: 0 auto;
+}
+
+.nav__container {
+  /* border: 1px solid red; */
+  list-style: none;
+  text-decoration: none;
+  /*text-align: center;*/
+  padding: 0;
+  margin: 0;
+  background: lightgray;
+  position: relative;
+}
+
+.nav__container,
+li,
+a {
+  display: inline-block;
+  padding: 10px;
+  text-decoration: none;
+  list-style: none;
+  font-size: 20px;
+  /* margin-left: 10px; */
+  margin: 0 auto;
+  color: gray;
+}
+
+.nav__container ul li {
+  text-decoration: none;
+}
+.logo {
+  /*border: 1px solid red;*/
+  display: inline-block;
+  /* border: 1px solid red; */
+  margin-left: 0px;
+  position: absolute;
+  margin-top: 20px;
+  margin-left: 50px;
+}
+
+.hero__container {
+  /* margin: 0 auto; */
+  width: 100%;
+  /* border: 1px solid red; */
+  display: inline-block;
+  margin: 0;
+  height: 550px;
+}
+
+.hero__heading {
+  display: inline-block;
+  font-size: 80px;
+  width: 430px;
+  margin: 48px auto;
+  padding: 10px 50px;
+  text-align: center;
+  /* border: 1px solid red; */
+  height: 400px;
+  vertical-align: top;
+  padding-left: 50px;
+}
+
+.line {
+  border-top: 1px solid black;
+  margin: 20px;
+}
+
+.hero__image {
+  display: inline-block;
+  width: 400px;
+  /* height: auto; */
+  /* border: 1px solid red; */
+  margin: 48px auto;
+  border-radius: 50%;
+  vertical-align: top;
+}
+
+.main__body {
+  width: 880px;
+  margin: 0 auto;
+}
+
+.btn {
+  border: 2px solid black;
+  width: 200px;
+  text-align: center;
+  box-shadow: 10px 5px 5px black;
+  
+}
+
+.middle-img {
+  margin: 10px;
+}
+
+.features__container {
+  /* border: 1px solid red; */
+  margin: 10px;
+  display: inline-block;
+}
+
+.features {
+  display: inline-block;
+  width: 48%;
+  margin: 0 10px;
+}
+
+.about {
+  display: inline-block;
+  width: 48%;
+}
+
+.services {
+  display: inline-block;
+  width: 31%;
+}
+
+.product {
+  display: inline-block;
+  width: 30%;
+}
+
```

```suggestion
.features {
  display: inline-block;
  width: 48%;
  margin: 0 10px;
}

.about {
  display: inline-block;
  width: 48%;
}

.services {
  display: inline-block;
  width: 31%;
}

.product {
  display: inline-block;
  width: 30%;
}
```
Awesome job on using percents thats very responsive